### ISSUE TITLE
K8ssandra operator deletes pvc on deletion

### ISSUE THREAD

**Issue by berginv** [k8ssandra/k8ssandra-operator#1229]
**What is missing?**

Preserve the pvcs when deleting a cluster

**Why do we need it?**

If there is an accidental deletion . The data will be lost forever .But if this feature exists. we can toggle whether to keep data or destroy altogether.

**Environment**

* K8ssandra Operator version:

    cr.k8ssandra.io/k8ssandra/k8ssandra-operator:v1.13.0

    
    

**Anything else we need to know?**:

**Comment by burmanm** [comment]
The data is preserved in PVs, not PVCs. k8ssandra-operator does not delete the PVs, that's controlled by your StorageClass.

**Comment by berginv** [comment]
Yes, you are correct . Please assume this scenario where I delete my k8ssandra cluster and recreate with no change. The operator will delete the PVCs and then recreate it, which will create a new PV (provisioned by storage class). In this scenario I can't use my existing PV. It would be good if there is a toggle as in this [Reference](https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-volume-claim-templates.html#k8s_controlling_volume_claim_deletion)

**Comment by xiongmozhou** [comment]
I really need this button, otherwise it would be troublesome every time I want to configure multiple storage and directories
